codespell_lib/data/dictionary.txt

```diff
@@ -22468,6 +22468,10 @@ rescrition->restriction
 rescritions->restrictions
 reseach->research
 reseached->researched
+researvation->reservation
+researvations->reservations
+researved->reserved
+researves->reserves
```

Comment:
Sorry @fishilico . I realised I missed the obvious one. Can you add the singular too please `researve->reserve`

Reply:
No problem. I did not know whether "researve" would be a false positive (Google shows some results for `"researve"`, including websites such as https://www.vivino.com/researve-unknown-v-7svbm/w/5990630 where it is not clear whether it is a misspelling).

On https://grep.app/, "researve" shows files that use "presearve", which seems to be a misspelling for "preserve". Therefore I added this word too.

I can squash my commits together, if you prefer.

Reply:
> No problem. I did not know whether "researve" would be a false positive (Google shows some results for `"researve"`, including websites such as https://www.vivino.com/researve-unknown-v-7svbm/w/5990630 where it is not clear whether it is a misspelling).

I'm pretty certain that's a typo too:
https://en.wikipedia.org/wiki/Reserve_wine

> On https://grep.app/, "researve" shows files that use "presearve", which seems to be a misspelling for "preserve". Therefore I added this word too.

Excellent, although you know what's coming next...

> I can squash my commits together, if you prefer.

I don't mind, indeed it's easier to review if you don't as we can see just what's changed. We can squash during the merge on GitHub if desired.